mis_builder/tests/test_mis_report_instance.py

```diff
@@ -484,25 +484,25 @@ def test_get_kpis_by_account_id(self):
             self.assertEqual(res[account_id], kpi200)
 
     def test_kpi_name_get_name_search(self):
-        r = self.env["mis.report.kpi"].name_search("k1")
+        r = self.env["mis.report.kpi"]._name_search("k1")
         self.assertEqual(len(r), 1)
-        self.assertEqual(r[0][0], self.kpi1.id)
-        self.assertEqual(r[0][1], "kpi 1 (k1)")
-        r = self.env["mis.report.kpi"].name_search("kpi 1")
+        self.assertEqual(r[0].id, self.kpi1.id)
+        self.assertEqual(r[0].display_name, "kpi 1 (k1)")
+        r = self.env["mis.report.kpi"]._name_search("kpi 1")
         self.assertEqual(len(r), 1)
-        self.assertEqual(r[0][0], self.kpi1.id)
-        self.assertEqual(r[0][1], "kpi 1 (k1)")
+        self.assertEqual(r[0].id, self.kpi1.id)
+        self.assertEqual(r[0].display_name, "kpi 1 (k1)")
 
     def test_kpi_expr_name_get_name_search(self):
-        r = self.env["mis.report.kpi.expression"].name_search("k1")
+        r = self.env["mis.report.kpi.expression"]._name_search("k1")
         self.assertEqual(
-            [i[1] for i in r],
+            [i.display_name for i in r],
             ["kpi 1 / subkpi 1 (k1.sk1)", "kpi 1 / subkpi 2 (k1.sk2)"],
         )
-        r = self.env["mis.report.kpi.expression"].name_search("k1.sk1")
-        self.assertEqual([i[1] for i in r], ["kpi 1 / subkpi 1 (k1.sk1)"])
-        r = self.env["mis.report.kpi.expression"].name_search("k4")
-        self.assertEqual([i[1] for i in r], ["kpi 4 (k4)"])
+        r = self.env["mis.report.kpi.expression"]._name_search("k1.sk1")
+        self.assertEqual([i.display_name for i in r], ["kpi 1 / subkpi 1 (k1.sk1)"])
+        r = self.env["mis.report.kpi.expression"]._name_search("k4")
+        self.assertEqual([i.display_name for i in r], ["kpi 4 (k4)"])
```

Comment:
I don't think these tests need to change in any way.

Reply:
These tests need to change cause previously `name_get` returned a tuple with both record id and name. Now its setted on display_name

Reply:
Actually, I'm pretty sure of what I'm saying here. Because the return type of `name_search` has not changed (it still returns a list of (id, display_name) tuples.  So can you please try without changing these two tests at all?

Reply:
Ok, done.